src/Storefront/Framework/Cache/CacheWarmer/Product/ProductRouteWarmer.php

```diff
@@ -29,7 +29,9 @@ public function __construct(IteratorFactory $iteratorFactory, ProductDefinition
     public function createMessage(SalesChannelDomainEntity $domain, ?array $offset): ?WarmUpMessage
     {
         $iterator = $this->iteratorFactory->createIterator($this->definition, $offset);
-        $iterator->getQuery()->setMaxResults(10);
+        $iterator->getQuery()
+            ->andWhere('`product`.active = 1')
```

Comment:
`active` is inheritable. You need to check parent too https://github.com/shopware/platform/blob/master/src/Core/Content/Product/ProductDefinition.php#L136

Reply:
Thanks for the feedback!

I have changed the code to check for both, the current and parent `active` flag.